app/src/main/java/com/example/criminalintentapp/presentation/fragments/crime_detail/CrimeFragment.kt

```diff
@@ -336,6 +349,7 @@ class CrimeFragment : Fragment(R.layout.fragment_crime), FragmentResultListener
         private const val REQUEST_DATE = "DialogDate"
         private const val TAG = "CrimeFragment"
         private const val DATE_FORMAT = "EEE, MMM, dd"
+        private const val AUTHORITY = "com.example.criminalintentapp.fileprovider"
```

Comment:
Now that you added this const here, you can go ahead and remove it from strings.xml